### ISSUE TITLE
Error: Cannot start event listening

### ISSUE THREAD

**Issue by Jonyder** [quentinlesceller/macmediakeyforwarder#10]
Hi,

I've just upgrade to the latest version. Though MacMediaKeyForwarder.app is activated under:
Privacy&Security > Accessibility
and 
Privacy&Security > Automation

When starting MacMediaKeyForwarder.app however it gives an error:
"Cannot start event listening. Please add Mac Media Key Forwarder to the "Security & Privacy" pane in System Preferences. Check "Accessibility" and "Automation" under the "Privacy" tab."

MacMediaKeyForwarder: 3.1.2
MacOS: 14.0 Beta (23A5286i)
Chip: Apple M2 Pro

**Comment by DaLuce2020** [comment]
Same behavior on Ventura 13.4, M1 Mac

**Comment by mgeauxblue** [comment]
Also occurring on Monterey 12.6.7, Intel MBP for both MacMediaKeyForwarder 3.1.2 and 3.1.1. Tried unchecking TCC Privacy entries > uninstalling app > reboot > reinstall and confirmed both TCC Privacy settings but still get the error.

Notably, v3.1 restored from TM backup runs fine and does not throw an error.

**Comment by nicholascarey** [comment]
Same. Latest versions of everything. 

Also supposedly this version is signed, but there is still the macOS  "Apple cannot check it for malicious software." warning on first opening. Have to do the option-click trick to even get past this.

**Comment by paleite** [comment]
The following fixed it for me: Go to Privacy & Security > Accessibility, then _remove_ MacMediaKeyForwarder from that list, and then re-add it. Then open MacMediaKeyForwarder and it will work again. Tested on v3.1.2

Edit: This was on Ventura 13.5, and I installed it via Homebrew.

**Comment by nicholascarey** [comment]
Yes, can confirm that works. 

Still the issue of it not being signed...

**Comment by DaLuce2020** [comment]
> The following fixed it for me: Go to Privacy & Security > Accessibility, then _remove_ MacMediaKeyForwarder from that list, and then re-add it. Then open MacMediaKeyForwarder and it will work again. Tested on v3.1.2

Might work on Monterey or lower, but not on Ventura

**Comment by nicholascarey** [comment]
> > The following fixed it for me: Go to Privacy & Security > Accessibility, then _remove_ MacMediaKeyForwarder from that list, and then re-add it. Then open MacMediaKeyForwarder and it will work again. Tested on v3.1.2
> 
> Might work on Monterey or lower, but not on Ventura

Worked for me on Ventura

**Comment by Jonyder** [comment]
> The following fixed it for me: Go to Privacy & Security > Accessibility, then _remove_ MacMediaKeyForwarder from that list, and then re-add it. Then open MacMediaKeyForwarder and it will work again. Tested on v3.1.2

Yes, thanks, worked for me as well with version 3.1.2 on MacOS Sonoma 14.0 Beta (23A5301h)

**Comment by quentinlesceller** [comment]
Marking this as solved.

**Comment by nicholascarey** [comment]
Has the issue about the app not being signed despite the release notes saying it is been addressed?

**Comment by quentinlesceller** [comment]
Hi, sorry did not notice the issue with signing. I'll investigate this for the next release.
